### DISCUSSION TITLE
Demucs-GUI_1.1_cuda closes upon immediately opening?

### DISCUSSION THREAD

**Discussion by Bakartridge** [CarlGao4/Demucs-Gui#54] [Q&A] [answered]
The title kinda says it, I installed the cuda version and extracted it, but whenever I try opening it the application just immediately closes, it opens up a python window for like a split second then it just closes. I have an AMD CPU (a Ryzen 9 7945HX) and an Nvidia GPU (RTX 4060), maybe it's just not compatible with AMD CPUs?

**Comment by CarlGao4** [comment]
Did you extract all the files to the same folder? After the terminal window closes, a starting window would show up a few seconds later

**Reply by Bakartridge** [reply]
Yeah it's all in the same folder, extracted it again to be safe and got the same result.

**Reply by CarlGao4** [reply]
Can you attach your latest log file? You can find them in `%appdata%\demucs-gui\logs`

**Reply by Bakartridge** [reply]
[20240325_230618_demucs_gui_log.log](https://github.com/CarlGao4/Demucs-Gui/files/14752315/20240325_230618_demucs_gui_log.log)

**Comment by CarlGao4** [comment, marked as answer]
Same error as https://github.com/CarlGao4/Demucs-Gui/issues/27#issuecomment-1768055897

**Reply by Bakartridge** [reply]
Perfect, that worked, thanks
